src/components/Gallery.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Movie from './Movie';
+import { getMovies } from '../util/swapi';
+import Header from './Header';
+import './Gallery.css';
+
+class Gallery extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
```

Comment:
Another good candidate for hooks refactoring :)

Here you will learn to use hook useEffect to replace componentDidMount.

Reply:
Gallery.js is now a functional component.